templates/overrides/menu/menu-local-tasks.html.twig

```diff
@@ -5,11 +5,15 @@
  */
 #}
 {% if primary %}
-  <h2 class="visually-hidden">{{ 'Primary tabs'|t }}</h2>
```

Comment:
A bit sad we lose this :(

Reply:
Can't remember why I removed it, so I put it back :|